src/GuiBase/BaseApplication.cpp

```diff
@@ -203,6 +204,9 @@ BaseApplication::BaseApplication( int argc, char** argv, const WindowFactory& fa
     createConnections();
     processEvents();
 
+    // Register the GeometrySystem converting loaded assets to meshes
+    m_engine->registerSystem( "GeometrySystem", new Ra::Engine::GeometrySystem, 1000 );
```

Comment:
use `std::limits<int>::max()` instead

Reply:
No, there might be cases where we want to be able to set a higher priority.

Reply:
really? like changing loading parameters?

Reply:
For instance. The thing is that we don't know, so why closing doors for no reason ?
Someone doing that is expected to know what he/she is doing.
Also it is in the application side, so no real impact on the library. We might need to update this if we add a management UI to load/unload plugins and adjust priorities in the main app (one day maybe).